appveyor.yml

```diff
@@ -46,7 +49,6 @@ test_script:
   - bundler --version
   - bash ./script/cibuild
 
-cache:
+#cache:
   # If one of the files after the right arrow changes, cache will be skipped
-  - C:\Ruby%RUBY_FOLDER_VER%\bin -> Gemfile,jekyll.gemspec,appveyor.yml
-  - C:\Ruby%RUBY_FOLDER_VER%\lib\ruby\gems\%GEMS_FOLDER_VER% -> Gemfile,jekyll.gemspec,appveyor.yml
+#  - 'vendor -> appveyor.yml,Gemfile,jekyll.gemspec'
```

Comment:
I'm reading http://bundler.io/v1.12/bundle_install.html and deployment seems the case for us.

We already have `without` https://github.com/jekyll/jekyll/blob/master/appveyor.yml#L19

So basically, we need to instruct bundle to install the gems in a custom folder which we can cache.